content/chainguard/chainguard-images/working-with-images/security-advisories/managing-advisories/index.md

```diff
@@ -0,0 +1,165 @@
+
+You can verify that you have successfully installed `wolfictl` by executing the `wolfictl version` command in your terminal, with output expected to be shown as follows. Note that the exact output will vary over time as new `wolfictl` versions are released. 
+You may need to add the `wolfictl` binary to your `$PATH` after installation if your system does not recognize the command.
+
+```shell
+wolfictl version
+ __        __   ___    _       _____   ___    ____   _____   _
+ \ \      / /  / _ \  | |     |  ___| |_ _|  / ___| |_   _| | |
+  \ \ /\ / /  | | | | | |     | |_     | |  | |       | |   | |
+   \ V  V /   | |_| | | |___  |  _|    | |  | |___    | |   | |___
+    \_/\_/     \___/  |_____| |_|     |___|  \____|   |_|   |_____|
+wolfictl: A CLI helper for developing Wolfi
+
+GitVersion:    devel
+GitCommit:     6c98dc69a559192575d085d87fd916d8281dd67d
+GitTreeState:  clean
+BuildDate:     2024-07-23T01:57:17
+GoVersion:     go1.22.5
+Compiler:      gc
+Platform:      darwin/arm64
+```
+
+
+## Cloning Package and Advisory Repositories
+
+We will need to clone a couple more repositories to continue: the `wolfi-dev/packages` repository and the `wolfi-dev/advisories` repository. Enter the following commands in your terminal to clone each of these repositories. We will also navigate to the `advisories` repository after cloning it.
+
+```shell
+git clone git@github.com:wolfi-dev/os.git 
+```
+
+```shell
+git clone git@github.com:wolfi-dev/advisories.git && cd advisories
+```
+
+
+## Viewing Existing Advisories
+
+Before we add to the database, we will take a look at the existing advisories issued for packages in Wolfi. Keep in mind that the results shown here, and on your own machine, are snapshots in time. You should regularly check for changes to the upstream repository as new packages and advisories are issued. 
+
+We will be using the `wolfi advisory list` command to view existing advisories. There are a variety of flags which we can append to assist in our search.
+`-p` lists all advisories for a given package name.
+`-V` lists all advisories for a given CVE ID, across all packages.
+`-t` lists all advisories by their most recent event type.
+`-c` lists all advisories by detected component type.
+`--history` reports the full list of events for displayed advisories.
+Multiple flags can be combined together to produce a more granular search. For a full listing of available flags, you can run the `wolfictl advisory list -h` command in your terminal.
+
+For example, let’s say you want to find advisories for the `glibc` package, and we want to see the full history of these advisories. Note that the output of this command may change as vulnerability entries are added and updated over time.
+
+```shell
+wolfictl advisory list -p glibc
+```
+
+The following shows a small sample of output from this command at the time of execution.
```

Comment:
```suggestion
The following shows a small sample of output from this command at the time of this writing. Note that the output of this command may change as vulnerability entries are added and updated over time.
```

"the time of execution" is a little subjective (like, the output I see is the output at the time of *my* execution). "the time of this writing" is clearer to me. Also, I think it makes sense to have these sentences right next to each other as they're speaking to the same idea